com.unity.render-pipelines.core/Runtime/Common/DynamicResolutionHandler.cs

```diff
@@ -544,5 +544,26 @@ public Vector2Int GetLastScaledSize()
         {
             return m_LastScaledSize;
         }
+
+        /// <summary>
+        /// Returns the resolved low res multiplier based on the low transparency threshold settings.
+        /// </summary>
+        /// <param name="targetLowRes"> the target low resolution.
+        ///     If by any chance thresholding is disabled or clamped, the exact same resolution is returned.
+        ///     This allows the caller to directly compare the float result safely with the floating point target resolution.
+        /// </param>
+        /// <returns>Returns the resolved low res multiplier based on the low transparency threshold settings.</returns>
+        public float GetLowResMultiplier(float targetLowRes)
+        {
+            if (!m_Enabled || !m_CachedSettings.enableLowResTransparencyThreshold)
+                return targetLowRes;
+
+            float thresholdPercentage = Math.Min(m_CachedSettings.lowResTransparencyMinimumThreshold / 100.0f, targetLowRes);
```

Comment:
Probably here we want to safe guard for lowResTransparencyMinimumThreshold == 0? Probably just early out if it's the case, otherwise looks like we could return a 0 resolution.

Reply:
if lowResTransparencyMinimumThreshold the condition at line 563 will catch it and just return the target low res. 
I will add a guard for the input instead. So you cant request a low res below some epsilon